include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -496,7 +496,7 @@ struct ModelMetadata : Base<OrtModelMetadata> {
 
   /** \brief Looks up a value by a key in the Custom Metadata map
    *
-   * \param zero terminated string key to lookup
+   * \param key zero terminated string key to lookup
    * \param allocator to allocate memory for the copy of the string returned
```

Comment:
```suggestion
   * \param allocator allocator to allocate memory for the copy of the string returned
```

Reply:
Yeah, we need to do a bigger pass if we want to fix all those...